ironfish/src/node.ts

```diff
@@ -168,8 +168,13 @@ export class IronfishNode {
     let workers = config.get('nodeWorkers')
     if (workers === -1) {
       workers = os.cpus().length - 1
+
+      const maxWorkers = config.get('nodeWorkersMax')
+      if (maxWorkers !== -1) {
```

Comment:
This is a nitpick, but I wonder if we should at least log if `nodeWorkers` is explicitly set to a value that's greater than `nodeWorkersMax`. It might be confusing to pass e.g. `--workers 8` and have it appear that nothing happens.

Reply:
By nothing happening, you mean that it doesn't cap it? Is the confusion that this only affects auto scaling? Like you would expect this to affect non auto scaling?

Reply:
Oh sorry, I misread it and thought it affected non-autoscaling. Looks good